opensearch_dashboards.json

```diff
@@ -15,5 +15,7 @@
     "dataSourceManagement"
   ],
   "server": true,
-  "ui": true
+  "ui": true,
+  "supportedOSDataSourceVersions": ">=1.0.0",
```

Comment:
I think we only tested with 1.3 + should we changes this to 1.3?

Reply:
@ZilongX Can you help answer this?

Reply:
1.3 check is for the managed side, for opensource side let's take in all supported OS versions